daemon/containerd/service.go

```diff
@@ -27,24 +28,25 @@ type ImageService struct {
 	containers      container.Store
 	snapshotter     string
 	registryHosts   RegistryHostsProvider
-	registryService RegistryConfigProvider
+	registryService RegistryService
 	eventsService   *daemonevents.Events
 }
 
 type RegistryHostsProvider interface {
 	RegistryHosts() docker.RegistryHosts
 }
 
-type RegistryConfigProvider interface {
+type RegistryService interface {
```

Comment:
FWIW; I looked if we should un-export these interfaces, as they're only _referenced_ locally, but I guess we exported them as they are used for the `ImageServiceConfig` below, and un-exporting them makes those fields awkward to use (and not show up in the documentation);

<img width="403" alt="Screenshot 2023-04-06 at 12 51 15" src="https://user-images.githubusercontent.com/1804568/230356083-ef7006a1-fa83-4698-945b-533963053bf8.png">